doc-src/install.html

```diff
@@ -43,17 +38,32 @@
 video/*; /usr/bin/open -Wn %s
 </pre>
 
+From now on you can run <a href="mitmproxy.html">mitmproxy</a> or <a href="mitmdump.html">mitmdump</a> from a terminal.
+
+## <a id=docUbuntu></a>Installation On Ubuntu
+
+Ubuntu comes with Python but we need to install pip, python-dev and several libraries.  This was tested on a fully patched installation of Ubuntu 14.04.
+
+<pre class="terminal">
+$ sudo apt-get python-pip python-dev libffi-dev libssl-dev libxml2-dev libxslt1-dev
+
+$ sudo pip install mitmproxy
+</pre>
+
+From now on you can run <a href="mitmproxy.html">mitmproxy</a> or <a href="mitmdump.html">mitmdump</a> from a terminal.
+
+## <a id=docWindows></a>Installation On Windows
 
-## Ubuntu
+Please note that mitmdump is the only component of mitmproxy that is supported on Windows.
 
-On Ubuntu, you will need the following native packages to install mitmproxy
-from source:
+Install the latest version of Python 2 from the <a href="https://www.python.org/downloads/windows/">Python website</a>.  The Windows MSI installer is fairly straightforward and even includes pip as long as you are installing a version of Python 2 after 2.7.9.  For the most part, you can accept the defaults but one option you should change during installation is "Add python.exe to Path".  This will make it easier to run Python.
+<img src="@!urlTo('winpythoninstaller.jpg')!@"/>
 
-- build-essential
-- python-dev
-- libffi-dev
-- libssl-dev
-- libxml2-dev
-- libxslt1-dev
+Now log out of Windows and back in to finish having Python added to your PATH.
```

Comment:
Terminal?  On Windows?  Not based on my testing, even did a "/taskkill /IM explorer.exe /F".  Couldn't find anything current on the issue, but [this](https://support.microsoft.com/en-us/kb/104011) says:
>However, note that modifications to the environment variables do not result in immediate change. For example, if you start another Command Prompt after making the changes, the environment variables will reflect the previous (not the current) values. The changes do not take effect until you log off and then log back on.